src/hooks/screen/useCreateChat.ts

```diff
@@ -82,6 +84,21 @@ const useCreateChat = () => {
     }
   };
 
+  const createAnonymousChat = async (selectedUser) => {
+    try {
+      setLoadingCreateChat(true);
+      const response = await getOrCreateAnonymousChannel(selectedUser?.user?.userId);
+      const chatData = createChannelJson(response, selectedUser);
+      const channelList = ChannelList.fromMessageAnonymouslyAPI(chatData);
+      await channelList.saveIfLatest(localDb);
+      handleMemberSchema(response);
+      goToChatScreen(channelList, GROUP_INFO);
+    } catch (e) {
+      setLoadingCreateChat(false);
+      console.log({e}, 'error create chat');
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `createAnonymousChat` does not handle errors properly. In case of an error, it only logs the error and sets `setLoadingCreateChat` to false. This might lead to silent failures where the user is not informed about the error. Consider adding error handling logic or at least show a generic error message to the user.
```diff
    } catch (e) {
        setLoadingCreateChat(false);
+       SimpleToast.show('An error occurred while creating the chat');
        console.log({e}, 'error create chat');
    }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->